[2022-05-23 20:15:39,106] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-02T00:00:00+00:00 [queued]>
[2022-05-23 20:15:39,117] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-02T00:00:00+00:00 [queued]>
[2022-05-23 20:15:39,118] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:39,118] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:15:39,118] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:39,135] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2022-01-02 00:00:00+00:00
[2022-05-23 20:15:39,141] {standard_task_runner.py:52} INFO - Started process 2171 to run task
[2022-05-23 20:15:39,144] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-01-02T00:00:00+00:00', '--job-id', '478', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpylgov030', '--error-file', '/tmp/tmp73p_k629']
[2022-05-23 20:15:39,145] {standard_task_runner.py:80} INFO - Job 478: Subtask second_task
[2022-05-23 20:15:39,204] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-01-02T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:15:39,300] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-02T00:00:00+00:00
[2022-05-23 20:15:39,301] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:15:39,301] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-05-23 20:15:39,315] {subprocess.py:85} INFO - Output:
[2022-05-23 20:15:39,316] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-05-23 20:15:39,317] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:15:39,343] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220102T000000, start_date=20220523T201539, end_date=20220523T201539
[2022-05-23 20:15:39,396] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:15:39,431] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
